Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/index.md

```diff
@@ -124,9 +124,12 @@ Made obsolete with the release of Umbraco v7.6 the media picker displays the cur
 ## Member Group Picker
 `Alias: Umbraco.MemberGroupPicker`
 
-## [Member Picker](Member-Picker.md)
+## [(Obsolete)Member Picker](Member-Picker.md)
 `Alias: Umbraco.MemberPicker`
 
+## [Member Picker]

```

Comment:
Any chance I could get you to remove the [] from the heading here? 😁

Reply:
Fixed in https://github.com/umbraco/UmbracoDocs/pull/1002/files#diff-6144233cd8a87f45eedd632df2632b9e 🦄